lib/FFI/Platypus/DL.pm

```diff
@@ -3,7 +3,7 @@ package FFI::Platypus::DL;
 use strict;
 use warnings;
 use 5.008004;
-use base qw( Exporter );
+use parent qw( Exporter );
```

Comment:
```suggestion
use Exporter qw( import );
```

I think this is probably better too, though we will want to make sure that Exporter is new enough in the prereqs.

Reply:
Sounds good to me! I'll update the commit

Reply:
Great looks like according to the Exporter documentation this was introduced in Perl 5.8.3

> Note: This feature was introduced in version 5.57 of Exporter, released with perl 5.8.3.

Since we dropped support for 5.8.3 some time ago we shouldn’t need to specify Exporter as a prereq.